expect.go

```diff
@@ -799,9 +799,14 @@ func (e *GExpect) ExpectSwitchCase(cs []Caser, timeout time.Duration) (string, [
 			}
 		case <-e.rcv:
 			// Data to fetch.
-			if _, err := io.Copy(&tbuf, e); err != nil {
+			nr, err := io.Copy(&tbuf, e)
+            if err != nil {
 				return tbuf.String(), nil, -1, fmt.Errorf("io.Copy failed: %v", err)
 			}
+            // now should reset timer??
```

Comment:
Could you expand the comment a bit. 

What does 0 signify and > 0 signify , why do we only need to reset the timer when it's larger than 0 ..

Reply:
timer should means  that, there was no new out put data in the past specific secdons. So timer shoud be reset when new out put is available.

Reply:
Yeah , just put that in the comment and I'll LGTM.

Reply:
PTAL ,（: